client/src/components/SponsorPage/SponsorsPage.jsx

```diff
@@ -129,11 +132,11 @@ class SponsorsPage extends React.Component {
       <Page title="Sponsors" {...this.props}>
         <FirstSectionContainer>
```

Comment:
maybe `PremiumSponsorsContainer` ? like on line 150 you have `CommunityContainer`

also they both should be with a symmetrical name `PremiumSponsorsContainer` and `CommunitySponsorsContainer`